lib/version.rb

```diff
@@ -16,8 +15,8 @@ module VERSION #:nodoc:
       MAJOR = PARTS[0].to_i
       MINOR = PARTS[1].to_i
       TINY = PARTS[2].to_i
-      PRE = PARTS[3]&.split("-", 2)&.first
-      DEV = PARTS[3]&.split("-", 2)&.second
```

Comment:
`Array#first` and `Array#second` are Rails methods but this module may be used before Rails is loaded.